networkx/algorithms/minors.py

```diff
@@ -9,22 +9,60 @@
 from networkx.exception import NetworkXException
 from networkx.utils import arbitrary_element
 
-__all__ = ["contracted_edge", "contracted_nodes", "identified_nodes", "quotient_graph"]
+__all__ = [
+    "contracted_edge",
+    "contracted_nodes",
+    "equivalence_classes",
+    "identified_nodes",
+    "quotient_graph"
+]
 
 chaini = chain.from_iterable
 
 
 def equivalence_classes(iterable, relation):
-    """Returns the set of equivalence classes of the given `iterable` under
-    the specified equivalence relation.
+    """Returns the set of equivalence classes, or blocks, of the equivalence
+    relation `relation` when applied to the elements of `iterable`.
```

Comment:
Sphinx wants a one line, short description followed by a blank line. It then uses this as a description of the function when another page links to this function.  After the blank line you can put all the details and alternate wording. Something like: 
```
"""Returns equivalence classes of `iterable` given by `relation`.

    The equivalence classes, or blocks, consist of objects from `iterable` which
    are all equivalent. They are defined to be equivalent if the `relation` function
    returns `True` when passed any two objects from that class, and `False` otherwise.
   To define an equivalence relation the function must be reflexive, symmetric and transitive.
```